ompi/mpi/man/man3/MPI_Bcast.md

```diff
@@ -0,0 +1,141 @@
+
+# NAME
+==================================
+
+**MPI_Bcast, MPI_Ibcast** - Broadcasts a message from the process with
+rank *root* to all other processes of the group.
+
+# SYNTAX
+==================================
+## C Syntax
+```C
+    #include <mpi.h>
+    int MPI_Bcast(void *buffer, int count, MPI_Datatype datatype,
+    	int root, MPI_Comm comm)
+
+    int MPI_Ibcast(void *buffer, int count, MPI_Datatype datatype,
+    	int root, MPI_Comm comm, MPI_Request *request)
+```
+## Fortran Syntax
+```Fortran
+    USE MPI
+    ! or the older form: INCLUDE 'mpif.h'
+    MPI_BCAST(BUFFER, COUNT, DATATYPE, ROOT, COMM, IERROR)
+    	<type>	BUFFER(*)
+    	INTEGER	COUNT, DATATYPE, ROOT, COMM, IERROR
+
+    MPI_IBCAST(BUFFER, COUNT, DATATYPE, ROOT, COMM, REQUEST, IERROR)
+    	<type>	BUFFER(*)
+    	INTEGER	COUNT, DATATYPE, ROOT, COMM, REQUEST, IERROR
+```
+## Fortran 2008 Syntax
+``` Fortran
+    USE mpi_f08
+    MPI_Bcast(buffer, count, datatype, root, comm, ierror)
+    	TYPE(*), DIMENSION(..) :: buffer
+    	INTEGER, INTENT(IN) :: count, root
+    	TYPE(MPI_Datatype), INTENT(IN) :: datatype
+    	TYPE(MPI_Comm), INTENT(IN) :: comm
+    	INTEGER, OPTIONAL, INTENT(OUT) :: ierror
+
+    MPI_Ibcast(buffer, count, datatype, root, comm, request, ierror)
+    	TYPE(*), DIMENSION(..), ASYNCHRONOUS :: buffer
+    	INTEGER, INTENT(IN) :: count, root
+    	TYPE(MPI_Datatype), INTENT(IN) :: datatype
+    	TYPE(MPI_Comm), INTENT(IN) :: comm
+    	TYPE(MPI_Request), INTENT(OUT) :: request
+    	INTEGER, OPTIONAL, INTENT(OUT) :: ierror
+```
+# INPUT/OUTPUT PARAMETERS
+==================================
+
+buffer
+
+:   Starting address of buffer (choice).
+
+count
+
+:   Number of entries in buffer (integer).
+
+datatype
+
+:   Data type of buffer (handle).
+
+root
+
+:   Rank of broadcast root (integer).
+
+comm
+
+:   Communicator (handle).
+
+# OUTPUT PARAMETERS
+==================================
+
+request
+
+:   Request (handle, non-blocking only).
+
+IERROR
+
+:   Fortran only: Error status (integer).
+
+# DESCRIPTION
+==================================
+
+MPI_Bcast broadcasts a message from the process with rank root to all
```

Comment:
I mentioned this below (I don't know offhand which order Github will show these comments): whenever an MPI function is referenced in the paragraph text, put it inside a paid of single quotes so that it renders as fixed-width text (in this case, `MPI_Bcast` instead of MPI_Bcast).